packages/lib/shim-init-node.js

```diff
@@ -186,36 +186,37 @@ function shimInit(options = null) {
 			const nativeImage = require('electron').nativeImage;
 			let image = nativeImage.createFromPath(filePath);
 			if (image.isEmpty()) throw new Error(`Image is invalid or does not exist: ${filePath}`);
-
 			const size = image.getSize();
 
-			let mustResize = size.width > maxDim || size.height > maxDim;
+			const canResize = size.width > maxDim || size.height > maxDim;
 
-			if (mustResize && resizeLargeImages === 'ask') {
-				const answer = shim.showMessageBox(_('You are about to attach a large image (%dx%d pixels). Would you like to resize it down to %d pixels before attaching it?', size.width, size.height, maxDim), {
+			let userAnswer;
```

Comment:
The user answer is now mutable, which doesn't make much sense. Please refactor to make it a constant again